middle_end/flambda/simplify/simplify_toplevel.rec.ml

```diff
@@ -20,16 +20,24 @@ open! Simplify_import
 
 let simplify_toplevel dacc expr ~return_continuation ~return_arity
       exn_continuation ~return_cont_scope ~exn_cont_scope =
+  let toplevel_cont = Continuation.create ~name:"toplevel" () in
```

Comment:
Maybe make it explicit that this is not a real thing by calling it `dummy_toplevel_continuation` and add a comment that this is only required by `Rec_uses`

Reply:
done